src/pages/study-room/hook/useAudio.ts

```diff
@@ -0,0 +1,10 @@
+import { useRef } from 'react';
+import sound from '../../../assets/audio/flipdish-ringer.mp3';
+const useAudio = () => {
```

Comment:
요런것도 개행 넣어주시면 좋을 것 같습니다~
```suggestion
import sound from '../../../assets/audio/flipdish-ringer.mp3';

const useAudio = () => {
```